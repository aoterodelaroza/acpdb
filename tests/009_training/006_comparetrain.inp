## check: 006_comparetrain.out -a1e-10
## delete: 006_comparetrain.db
## delete: 006_comparetrain-set.h2o_h2o_1.xyz
## delete: 006_comparetrain-set.h2o_h2o_2.xyz
## delete: 006_comparetrain-set.h2o_h2o.xyz
## delete: 006_comparetrain-set.nh3_nh3_1.xyz
## delete: 006_comparetrain-set.nh3_nh3_2.xyz
## delete: 006_comparetrain-set.nh3_nh3.xyz
## labels: regression quick

verbose
system rm -f 006_comparetrain.db
connect 006_comparetrain.db

insert method m_empty
end
insert method m_ref
end
insert set s22b
 din ../zz_source/s22partial.din
 directory ../zz_source/s22_xyz
 method m_ref
end
insert set s22b-b
 din ../zz_source/s22partial.din
 directory ../zz_source/s22_xyz
 method m_ref
end
insert evaluation
 method m_empty
 property s22b.nh3_nh3
 value 0.2
end
insert evaluation
 method m_empty
 property s22b.h2o_h2o
 value 0.3
end
insert term
 method m_empty
 property s22b.h2o_h2o
 atom H
 L 0
 exponent 0.1
 value 0.3
end
insert term
 method m_empty
 property s22b.h2o_h2o
 atom H
 L 0
 exponent 0.2
 value 0.3
end
insert term
 method m_empty
 property s22b.nh3_nh3
 atom H
 L 0
 exponent 0.1
 value 0.3
end
insert term
 method m_empty
 property s22b.nh3_nh3
 atom H
 L 0
 exponent 0.2
 value 0.3
end

training
 atom H l
 exp 0.1 0.2
 empty m_empty
 reference m_ref
 subset
  set s22b
 end
end

compare
 source ../zz_source/s22b.list
 property_type energy_difference
 method m_ref
 training s22b
end
compare
 source ../zz_source/s22b.list
 property_type energy_difference
 method m_ref
 training
end
