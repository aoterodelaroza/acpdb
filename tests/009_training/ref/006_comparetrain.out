%% verbose
%% system rm -f 006_comparetrain.db
* SYSTEM: rm -f 006_comparetrain.db

%% connect 006_comparetrain.db
* CONNECT 

Disconnecting previous database (if connected) 
Connecting database file 006_comparetrain.db
Creating skeleton database 

%% insert method m_empty
* INSERT: insert data into the database (METHOD)
# INSERT METHOD m_empty

%% insert method m_ref
* INSERT: insert data into the database (METHOD)
# INSERT METHOD m_ref

%% insert set s22b
* INSERT: insert data into the database (SET)
# INSERT SET s22b
# INSERT STRUCTURE s22b.nh3_nh3
# INSERT STRUCTURE s22b.nh3_nh3_1
# INSERT STRUCTURE s22b.nh3_nh3_2
# INSERT PROPERTY s22b.nh3_nh3
# INSERT EVALUATION (method=m_ref;property=s22b.nh3_nh3)
# INSERT STRUCTURE s22b.h2o_h2o
# INSERT STRUCTURE s22b.h2o_h2o_1
# INSERT STRUCTURE s22b.h2o_h2o_2
# INSERT PROPERTY s22b.h2o_h2o
# INSERT EVALUATION (method=m_ref;property=s22b.h2o_h2o)

%% insert set s22b-b
* INSERT: insert data into the database (SET)
# INSERT SET s22b-b
# INSERT STRUCTURE s22b-b.nh3_nh3
# INSERT STRUCTURE s22b-b.nh3_nh3_1
# INSERT STRUCTURE s22b-b.nh3_nh3_2
# INSERT PROPERTY s22b-b.nh3_nh3
# INSERT EVALUATION (method=m_ref;property=s22b-b.nh3_nh3)
# INSERT STRUCTURE s22b-b.h2o_h2o
# INSERT STRUCTURE s22b-b.h2o_h2o_1
# INSERT STRUCTURE s22b-b.h2o_h2o_2
# INSERT PROPERTY s22b-b.h2o_h2o
# INSERT EVALUATION (method=m_ref;property=s22b-b.h2o_h2o)

%% insert evaluation
* INSERT: insert data into the database (EVALUATION)
# INSERT EVALUATION (method=m_empty;property=s22b.nh3_nh3)

%% insert evaluation
* INSERT: insert data into the database (EVALUATION)
# INSERT EVALUATION (method=m_empty;property=s22b.h2o_h2o)

%% insert term
* INSERT: insert data into the database (TERM)
# INSERT TERM (method=m_empty;property=s22b.h2o_h2o;atom=H;l=0;exponent=0.1)

%% insert term
* INSERT: insert data into the database (TERM)
# INSERT TERM (method=m_empty;property=s22b.h2o_h2o;atom=H;l=0;exponent=0.2)

%% insert term
* INSERT: insert data into the database (TERM)
# INSERT TERM (method=m_empty;property=s22b.nh3_nh3;atom=H;l=0;exponent=0.1)

%% insert term
* INSERT: insert data into the database (TERM)
# INSERT TERM (method=m_empty;property=s22b.nh3_nh3;atom=H;l=0;exponent=0.2)

%% training
* TRAINING: started defining the training set 

%% atom H l
%% exp 0.1 0.2
%% empty m_empty
%% reference m_ref
%% subset
%% end
* TRAINING: fininshed defining the training set 

## Description of the training set
# List of atoms and maximum angular momentum channels (1)
| Atom | lmax |
| H | l |

# List of exponents (2)
| id | exp |
| 0 | 0.1 |
| 1 | 0.2 |

# List of subsets (1)
| id | alias | db-name | db-id | initial | final | size | dofit? | litref | description |
| 0 | s22b | s22b | 1 | 1 | 2 | 2 | 1 |  |  |

# List of methods
| type | name | id | for fit? |
| reference | m_ref | 2 | n/a |
| empty | m_empty | 1 | n/a |

# List of properties (2)
| fit? | id | property | propid | alias | db-set | proptype | nstruct | weight | refvalue |
| yes | 1 | s22b.nh3_nh3 | 1 | s22b | s22b | ENERGY_DIFFERENCE | 3 | 1 | -3.133000 |
| yes | 2 | s22b.h2o_h2o | 2 | s22b | s22b | ENERGY_DIFFERENCE | 3 | 1 | -4.989000 |

# Calculation completion for the current training set
# Reference: 2/2 (complete)
# Empty: 2/2 (complete)
# Terms: 
| H | l | 0.1 | 2/2 | (complete)
| H | l | 0.2 | 2/2 | (complete)
# Total terms: 4/4 (complete)

%% compare
* COMPARE: compare data to database evaluations

# -- Evaluation of data from file -- 
# File: ../zz_source/s22b.list
# Property type: ENERGY_DIFFERENCE
# Reference method: m_ref
# Statistics: 
# s22b             rms = 3573.19230892   mae = 3447.24108500   mse = -3447.24108500  wrms = 5053.25702425  ndat = 2
# ALL              rms = 3573.19230892   mae = 3447.24108500   mse = -3447.24108500  wrms = 5053.25702425  ndat = 2
#
Id Name                                                File         Ref_method         difference
1 s22b.nh3_nh3                              -2510.0378800000      -3.1330000000   -2506.9048800000
2 s22b.h2o_h2o                              -4392.5662900000      -4.9890000000   -4387.5772900000

%% compare
* COMPARE: compare data to database evaluations

# -- Evaluation of data from file -- 
# File: ../zz_source/s22b.list
# Property type: ENERGY_DIFFERENCE
# Reference method: m_ref
# Statistics: 
# s22b             rms = 3573.19230892   mae = 3447.24108500   mse = -3447.24108500  wrms = 5053.25702425  ndat = 2
# ALL              rms = 3573.19230892   mae = 3447.24108500   mse = -3447.24108500  wrms = 5053.25702425  ndat = 2
#
Id Name                                                File         Ref_method         difference
1 s22b.nh3_nh3                              -2510.0378800000      -3.1330000000   -2506.9048800000
2 s22b.h2o_h2o                              -4392.5662900000      -4.9890000000   -4387.5772900000

